### PR TITLE
Add checker for lp1812361

### PR DESCRIPTION
**Pull request by sombrafam**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -7,3 +7,4 @@ snap/.snapcraft
 .tox
 *.pyc
 .stestr
+.idea
```
Comment (dosaboy) [inline .gitignore:10]: what is this?

File: defs/scenarios/juju/charms_bugs.yaml [hunk 2]
```diff
@@ -3,6 +3,12 @@ checks:
     input:
       path: 'var/log/juju/unit-ceph-osd*.log'
     expr: '.* Cannot zap a device used by lvm'
+
```
Comment (dosaboy) [inline defs/scenarios/juju/charms_bugs.yaml:6]: can we lose the newline chars either side of this please?

File: defs/scenarios/juju/charms_bugs.yaml [hunk 3]
```diff
@@ -14,3 +20,15 @@ conclusions:
         failed since the disk has an LVM header. If you are sure the disk is
         not in active use by LVM, you may run pvremove on it before using it as
         an OSD - see LP bug for details.
+
```
Comment (dosaboy) [inline defs/scenarios/juju/charms_bugs.yaml:23]: again with the newline chars

File: defs/scenarios/juju/charms_bugs.yaml [hunk 4]
```diff
@@ -14,3 +20,15 @@ conclusions:
...
+  lp1812361:
+    decision: has_1812361
+    raises:
+      type: LaunchpadBug
+      bug-id: 1812361
+      message: >-
+        known glance-simplestream-sync bug identified - this happens because
+        the relation data received from keystone does not have the expected
+        data. Upgrading charm-keystone to version >= stable/539 should fix the
```
Comment (dosaboy) [inline defs/scenarios/juju/charms_bugs.yaml:32]: Can you add some information to the message about what information is missing from the relation that is expected to exist?

**Review by dosaboy — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (sombrafam) [reply .gitignore:10]: Pycharm IDE config file. Like this: https://github.com/openstack/nova/blob/master/.gitignore#L20

File: defs/scenarios/juju/charms_bugs.yaml (hunk 2 above)
Comment (sombrafam) [reply defs/scenarios/juju/charms_bugs.yaml:6]: Not sure what you mean by that. Most checks are like this. Do you have an example?

File: defs/scenarios/juju/charms_bugs.yaml (hunk 4 above)
Comment (sombrafam) [reply defs/scenarios/juju/charms_bugs.yaml]: Done

**Comment by dosaboy** [comment]
lgtm thanks erlon!

**Pull request merged**